blocks/career-carousel/career-carousel.js

```diff
@@ -0,0 +1,158 @@
+  let curActive;
+  for (let i = 0; i < spans.length; i += 1) {
+    if (spans[i].classList.contains('active-nav')) {
+      curActive = i;
+      break;
+    }
+  }
+
+  if (curActive === undefined) {
+    return;
+  }
+
+  let newActive = curActive + (next ? 1 : -1);
+  if (newActive < 0) {
+    newActive = spans.length - 1;
+  } else if (newActive >= spans.length) {
+    newActive = 0;
+  }
+
+  if (slideDivs.length <= newActive) {
+    return;
+  }
+
+  scrollToCard(
+    newActive,
+    slideDivs[newActive],
+    newActive > 0 ? slideDivs[newActive - 1] : null,
+    slides,
+    spans[newActive],
+    doc,
+  );
+}
+
+export default async function decorate(block) {
+  const json = await fetchIndex('query-index', 'career-testimonials');
+  const data = filterIncompleteEntries(json);
+
+  const careerSlider = document.createElement('div');
+  careerSlider.classList.add('career-slider');
+
+  const careerSlides = document.createElement('div');
+  careerSlides.classList.add('career-slides');
+
+  const slideDivs = [];
+  for (let i = 0; i < data.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('career-card');
+    const a = document.createElement('a');
+    a.href = data[i].path;
+    const pic = createOptimizedPicture(data[i].image, data[i].pagename);
+    a.append(pic);
+
+    const bqc = document.createElement('div');
+    bqc.classList.add('career-card-bqc');
+    const bq = document.createElement('blockquote');
+    bq.textContent = data[i]['career-quote'];
+    bqc.append(bq);
+    a.append(bqc);
+
+    const nm = document.createElement('h6');
+    nm.textContent = data[i].pagename;
+    a.append(nm);
+
+    const role = document.createElement('p');
+    role.textContent = data[i]['career-jobtitle'];
+    a.append(role);
+
+    const link = document.createElement('button');
+    link.textContent = 'Read More '; // TODO
+    const arrow = document.createElement('img');
+    arrow.src = '/icons/angle-right-blue.svg';
+    arrow.alt = 'Go to testimonial';
+    arrow.classList.add('icon-angle-right-blue');
+    link.append(arrow);
+    a.append(link);
+    div.append(a);
+
+    careerSlides.append(div);
+    slideDivs.push(div);
+  }
+  careerSlider.append(careerSlides);
+
+  const navBar = document.createElement('div');
+  navBar.classList.add('career-slides-navbar');
+  const navButtons = document.createElement('div');
+  navButtons.classList.add('career-slides-nav');
+
+  const buttons = [];
+  for (let i = 0; i < data.length; i += 1) {
+    const prevDiv = i > 0 ? slideDivs[i - 1] : null;
+
+    const s = document.createElement('span');
+    if (i === 0) {
+      s.classList.add('active-nav');
+    }
+    s.tabIndex = '-1';
+    s.onclick = () => scrollToCard(i, slideDivs[i], prevDiv, careerSlides, s, document);
+    navButtons.append(s);
+
+    buttons.push(s);
+  }
+  const la = document.createElement('img');
+  la.src = '/icons/angle-left-blue.svg';
```

Comment:
Can be in css

Reply:
I guess it can be but then it has to be changed to a background image, these arrows aren't really background images... AFAIK there is no portable way to set the img src tag in CSS yet.